src/lib/rc/dsm.cpp

```diff
@@ -470,6 +470,7 @@ dsm_decode(hrt_abstime frame_time, uint16_t *values, uint16_t *num_values, bool
 		 * Specifically, the first four channels in rc_channel_data are roll, pitch, thrust, yaw,
 		 * but the first four channels from the DSM receiver are thrust, roll, pitch, yaw.
 		 */
+#if !defined(__PX4_QURT)
```

Comment:
Don't know why this mapping was introduced. When I test with eagle board, I had to disable this mapping to make it work.